test/test_unitary.py

```diff
@@ -84,3 +84,38 @@ def test_compute_gates(self):
         calc2 = v_gate @ np.diag(diag).conj().T @ w_gate
         self.assertTrue(np.allclose(calc1, gate1))
         self.assertTrue(np.allclose(calc2, gate2))
+
+    def test_compute_gates_fixed(self):
+        """ test auxiliar funciont compute gates"""
+        # Matrices gate1 and gate2 are not unitary.
```

Comment:
```suggestion

```

`qclib.unitary._is_unitary(gate1)` and `qclib.unitary._is_unitary(gate2)` return `True`